STARTERKIT/gulpfile.js

```diff
@@ -38,7 +38,9 @@ options.theme = {
   build : options.rootPath.theme + 'components/asset-builds/',
   css : options.rootPath.theme + 'components/asset-builds/css/',
   js : options.rootPath.theme + 'js/',
-  node : options.rootPath.theme + 'node_modules/'
+  node : options.rootPath.theme + 'node_modules/',
+  images     : options.rootPath.theme + 'images/',
+  sprites    : 'images-source/*'
```

Comment:
Would be better to use full path as in the rest of the attributes 
`sprites     : options.rootPath.theme + 'images-source/*'`